build.gradle

```diff
@@ -48,8 +48,8 @@ dependencies {
 
 	testCompile "com.github.tomakehurst:wiremock-jre8:2.32.0"
 
-	testImplementation "org.junit.jupiter:junit-jupiter:5.7.1"
-	testRuntimeOnly('org.junit.jupiter:junit-jupiter-engine:5.7.1')
+	testImplementation "org.junit.jupiter:junit-jupiter:5.9.0"
```

Comment:
Just because or it helped fix something ?

Reply:
Yup, I don't know what execution order JUnit uses by default but I can't reliably reproduce a problem that has to do with the order tests are run. With the latest JUnit you can tell it what ordering to use.